opentelemetry-sdk/src/trace/sampler/jaeger_remote/sampler.rs

```diff
@@ -203,7 +203,13 @@ impl JaegerRemoteSampler {
                     // send request
                     match Self::request_new_strategy(&client, endpoint.clone()).await {
                         Ok(remote_strategy_resp) => strategy.update(remote_strategy_resp),
-                        Err(err_msg) => global::handle_error(TraceError::Other(err_msg.into())),
+                        Err(err_msg) => {
+                            otel_debug!(
+                                name: "JaegerRemoteSampler.UpdateStrategy.RequestFailed",
+                                message = "Failed to fetch new sampling strategy",
```

Comment:
not sure what is the impact of this for end users. depending on the impact, we may need a higher severity.

Reply:
Have raised to otel_warn - this could be actionable for the enduser as the remote endpoint is not available. The impact is that the last fetched sampling configuration will be used till success.

Reply:
```rust
 otel_warn!(
      name: "JaegerRemoteSampler.UpdateStrategy.RequestFailed",
       message = "Failed to fetch the sampling strategy from the remote endpoint. The last successfully fetched
 configuration will be used if available; otherwise, the default sampler will be applied until a successful 
configuration fetch.",
       reason = format!("{}", err_msg),
);
```